src/resolvers/profileResolver.ts

```diff
@@ -0,0 +1,53 @@
+import { Profile } from '../models/user'
+import cloudinary from 'cloudinary';
+import path from 'path';
+
+cloudinary.config({
```

Comment:
Remember to add these values in the `.env.example`, I mean the names, not actually credentials